src/graph/graph.c

```diff
@@ -1063,25 +1021,24 @@ uint Graph_GetNodeLabels
 	GrB_Info res;
 	UNUSED(res);
 
-	// GrB_Col_extract will iterate over the range of the output size
-	RG_Matrix M = Graph_GetNodeLabelMatrix(g);
+	Delta_Matrix M = Graph_GetNodeLabelMatrix(g);
 
 	EntityID id = ENTITY_GET_ID(n);
-	RG_MatrixTupleIter iter = {0};
-	res = RG_MatrixTupleIter_AttachRange(&iter, M, id, id);
+	Delta_MatrixTupleIter iter = {0};
+	res = Delta_MatrixTupleIter_AttachRange(&iter, M, id, id);
 	ASSERT(res == GrB_SUCCESS);
 
 	uint i = 0;
 
 	for(; i < label_count; i++) {
 		GrB_Index col;
-		res = RG_MatrixTupleIter_next_BOOL(&iter, NULL, &col, NULL);
+		res = Delta_MatrixTupleIter_next_BOOL(&iter, NULL, &col, NULL);
```

Comment:
Another approach would be to extract the row and then use `GxB_Vector_export_CSC` to get the column indices.